review/tests.py

```diff
@@ -2,52 +2,49 @@
+    def test_persist_review(self, review_fixture, review_data_fixture):
+        assert review_fixture.review == pytest.REVIEW
+        assert review_fixture in Review.objects.all()
 
-    def test_rating_limit_min(self):
-        with pytest.raises(ValidationError):
-            Review.create(review=REVIEW, rating=-1, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+    def test_del_review(self, review_fixture):
+        review_fixture.delete()
+        assert review_fixture not in Review.objects.all()
 
-    def test_rating_limit_max(self):
+    @pytest.mark.parametrize("rating", [-1, 6])
+    def test_rating_limit_min(self, rating, review_data_fixture):
         with pytest.raises(ValidationError):
-            Review.create(review=REVIEW, rating=6, daycare_id=DAY_CARE_ID, dogowner_id=DOG_OWNER_ID)
+            Review.create(review=pytest.REVIEW, rating=rating, daycare_id=pytest.DAY_CARE_ID,
+                          dogowner_id=pytest.DOG_OWNER_ID)
 
-    def test_delete_related_daycare(self, test_review):
-        daycare = DayCare.objects.get(pk=DAY_CARE_ID)
+    def test_delete_related_daycare(self, review_fixture, review_data_fixture):
+        daycare = DayCare.objects.get(pk=pytest.DAY_CARE_ID)
 
         assert daycare in DayCare.objects.all()
-        assert test_review in Review.objects.all()
+        assert review_fixture in Review.objects.all()
 
         daycare.delete()
         assert daycare not in DayCare.objects.all()
-        assert test_review not in Review.objects.all()
-
-    def test_delete_related_dogowner(self, test_review):
-        dogowner = DogOwner.objects.get(id=DOG_OWNER_ID)
-
-        assert dogowner in DogOwner.objects.all()
-        assert test_review in Review.objects.all()
-
-        dogowner.delete()
-        assert dogowner not in DogOwner.objects.all()
-        assert test_review in Review.objects.all()
+        assert review_fixture not in Review.objects.all()
+
+    def test_delete_related_dogowner(self, dogowner_fixture, review_fixture, review_data_fixture):
+        assert dogowner_fixture in DogOwner.objects.all()
+        assert review_fixture in Review.objects.all()
+
+        dogowner_fixture.delete()
+        assert dogowner_fixture not in DogOwner.objects.all()
+        assert review_fixture in Review.objects.all()
+
+    @pytest.mark.parametrize("number_of_reviews", [0, 1, 3])
+    def test_get_review_by_daycare_id(self, number_of_reviews, dogowner_fixture, daycare_fixture):
+        review_list = []
+        for i in range(1, number_of_reviews + 1):
+            review_list.append(Review.create(review=f'review {i}', rating=i, daycare_id=daycare_fixture.id,
+                                             dogowner_id=dogowner_fixture.id))
+        reviews = Review.get_review_by_daycare_id(daycare_fixture.id)
+        assert len(reviews) == number_of_reviews
+
+        for position, review in enumerate(review_list):
+            assert review == review_list[position]
```

Comment:
On a second look, I don't get this assertion. why are we checking that `review_list[n]==review_list[n]`?

Reply:
lol nice catch!
fixed